app/customer.py

```diff
@@ -0,0 +1,26 @@
+from app.car import Car
+
+
+class Customer:
+    def __init__(
+        self,
+        name: str,
+        product_cart: dict,
+        location: list,
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]